CONTRIBUTING.md

```diff
@@ -8,15 +8,16 @@ Fork, then clone the repo:
 git clone git@github.com:your-username/go-grpc-middleware.git
 ```    
 Before submitting a patch, please make sure to run the following make commands for running checks - 
-Make commands for running formatting/tests/generate proto files/vetting
+Before submitting a patch, please make sure to run the following make commands for running
+formatting check, regenerate proto files, and run tests and linters:
```

Comment:
Don't forget to delete the line above this as well.
```suggestion
Before submitting a patch, please make sure to run the following make commands to run the
formatting check, regenerate the proto files, and run the tests and linters:
```

Reply:
removed the duplicated line; RE suggestion - (nit) too many verbs `run` in one sentence, how about this (in 7aa9f85):
```
Before submitting a patch, please make sure to run the following make commands to execute the
formatting check, regenerate the proto files, and run the tests and linters:
```